__tests__/app.test.js

```diff
@@ -89,6 +89,42 @@ describe("GET - /api/articles/invalidArticleId", () => {
       .expect(404)
       .then((response) => {
         expect(response.body.msg).toBe("not found");
+
+      });
+  });
+});
+
+describe.only("/api/articles", () => {
```

Comment:
- So i fixed the first point so checking the article_id tobe 1;
- i  fixed the second bullet point 
- i don't know what i need to do for this bullet point because i have the logic in the model.js that if the length is 0 ew reject the promises and send back a custom error 

the .only appear only in the main branch and not in the ncnews6